addon/mixins/application-route-mixin.js

```diff
@@ -58,6 +58,10 @@ export default Mixin.create({
     return fastboot ? fastboot.get('isFastBoot') : false;
   }),
 
+  _cookies: computed(function() {
+    return getOwner(this).lookup('service:cookies');
+  }),
```

Comment:
We should lookup the service inside of the `sessionAuthenticated` method so that we don't add any unnecessary properties via the mixin.